### DISCUSSION TITLE
Upgrade to 1.0.182 causing view app issue

### DISCUSSION THREAD

**Discussion by gpjorgenson** [Budibase/budibase#6077] [Help] [answered]
I upgraded to 1.0.182 hoping it would correct the issue I delineated in discussion #5808. However it brought a new issue. When I click on 'view app' , it opens the last app I published regardless of which app I clicked 'view app' on. Example: I published "InternalCustServ" and did a 'view app', then I went in and did a 'view app' on "registration demo". Instead of viewing "registration demo", it displayed "InternalCusServ" ... Please note the URL's in the provided screen clips
![budibase-clip01](https://user-images.githubusercontent.com/23404511/170540640-c98a8068-f344-425e-9db1-196b0cef514c.png)
![budibase-clip02](https://user-images.githubusercontent.com/23404511/170540669-6be1cf42-1ee5-4549-85fe-9f15ef5ec3ac.png)
.

**Comment by melohagan** [comment]
Hi @gpjorgenson 

I can't reproduce this. Tested on v1.0.185

**Comment by gpjorgenson** [comment, marked as answer]
That is unfortunate. It is consistent for me. And any other user still gets {"message":"Missing/invalid DB name","status":500}... The users are all defined in the budibase db. When I tried to delete a user in the general users budibase db and then redo it from scratch, the users displayed in the app were flagged inactive and a new one with the same email address showed up in the apps. This issue has brought our budibase projects to a standstill.  
For now, all other tables are remote postgresql. But we have plans to connect to LDAP's if we ever get past this user issue.

**Reply by mike12345567** [reply]
Hi @gpjorgenson - I cannot re-create either of these issues, I'm wondering if something has gone wrong during the upgrade process.

Out of interest how did you upgrade from 1.0.8x to the latest version? From such an old version you will likely need to update your `docker-compose.yaml` and `.env` files, or your Chart files depending on self hosting mechanism.

As a test, if you setup a completely separate Budibase self hosted install, ideally using the CLI, (download [here](https://github.com/Budibase/budibase/releases/tag/v1.0.184)) then try importing your app to it, as well as setting up a user in it to check if its an environmental/installation problem - if this still doesn't work then we might need an export of your app to debug further.

**Reply by gpjorgenson** [reply]
I have just been clicking on the upgrade menu option and have done so whenever I see an available upgrade. I will pursue this suggestion with my docker expert and see what happens. He is pretty busy atthe moment so it might take a few days.

**Reply by mike12345567** [reply]
Hi @gpjorgenson - no problem, I expect your issue is your docker-compose file needing updated in that case, the Update button within the UI should work consistently from now on, but we had to make a few changes to Budibase which means that not updating your docker-compose file can cause problems under the hood!

**Reply by gpjorgenson** [reply]
I was able to get my programmer to correctly update budibase and docker to the current level 1.0.182. That does appear to have fixed the issues I was having. What seems to have happened to cause the issues is I clicked on the 'Check for updates' on our standalone installation and it indicated that it had successfully completed the update. So I updated the apps. Started having problems after that. And an app reversion did not help. So I am speculating that the update was not clean for some reason and our install's user controls and app viewer became partially corrupted in some way. No proof, just speculation.

**Reply by gpjorgenson** [reply]
I see you did do some adjustments to the update button. But I think I will still let my programmer manage that from now on. Thanks for your response and help.

**Reply by gpjorgenson** [reply]
My apologies... Looks like my programmer actually updated us to version 1.0.194. Sorry for getting that wrong up above.
